point/chain.json

```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "point",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "Point",
```

Comment:
Could this change to 'Point Network' please? so it will then match the SLIP173 registration

Reply:
Done